### PR TITLE
Fix one more HA scene problem

### PR DESCRIPTION
**Pull request by ultratoto14**
I create a bunch of scenes with different kind of device configuration:
- RGB + CCT
- RGB + W
- RGB
- CCT
- W dimmer
Found a case that needs this fix 😄

### CODE REVIEW

File: custom_components/localtuya/light.py [hunk 1]
```diff
@@ -188,7 +188,9 @@ def hs_color(self):
         """Return the hs color value."""
         if self.is_color_mode:
             return self._hs
-        if self.supported_features & SUPPORT_COLOR:
+        if self.supported_features & SUPPORT_COLOR and not self.has_config(
```
Comment (postlund) [inline custom_components/localtuya/light.py:191]: I think you can do this:

```python
if self.supported_features & (SUPPORT_COLOR | SUPPORT_COLOR_TEMP):
```

Comment (ultratoto14) [reply custom_components/localtuya/light.py:191]: It's a `not`,  I mean, in case SUPPORT_COLOR but does not SUPPORT_COLOR_TEMP

Comment (postlund) [reply custom_components/localtuya/light.py:191]: Ah, right, missed that. But you should be able to use `not. self.supported_features & SUPPORT_COLOR_TEMP` for consistency.

Comment (ultratoto14) [reply custom_components/localtuya/light.py:191]: Ok

**Review by postlund — APPROVED** [review]

**Pull request merged**